ic-os/guestos/context/docker-base.prod

```diff
@@ -1 +1 @@
-ghcr.io/dfinity/guestos-base@sha256:0a485d2d033df5aa7efd4ccbaad432c29d18f0f165cdcbeed5d49790c0d16340
+ghcr.io/dfinity/guestos-base@sha256:4cdcde6203e0b605467230826e7dae717a7320803f1cad1f102cbed94d7f1de7
```

Comment:
This contains changes for the GuestOS 24.04 upgrade, we'll keep this in an RC for the initial rollout